react/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Is this line necessary? I can't think why it would be 🤔

Reply:
I believe some IDE plugin put it automatically. 
Removing it